crates/driver/openapi.yml

```diff
@@ -436,6 +437,24 @@ components:
           description: The factor of the user surplus that the protocol will request from the solver after settling the order
           type: number
           example: 0.5
+    PriceImprovement:
+      description: |
+        If the order was out of market during creation, pay the protocol a factor of the difference.
```

Comment:
`A cut from the price improvement over the best quote is taken as a protocol fee.`